backend/fms_core/viewsets/_constants.py

```diff
@@ -249,6 +249,13 @@
     "validation_status_timestamp": DATE_FILTERS,
 }
 
+_readset_filterset_fields: FiltersetFields = {
+    **_prefix_keys("dataset__", _dataset_filterset_fields),
```

Comment:
Put the model own fields first in the filterset. Move this at the end of the list ...